### PR TITLE
Hotfix/1.0.0 beta9

### PR DESCRIPTION
**Pull request by perruche**
## Update
- @studiometa/js-toolkit to v2.0.0-beta.0
- Update JS files in the theme to match js-toolkit v2
- npm dependencies (eslint, prettier, babel, tailwindcss)

## Added
- Individual @studiometa config packages (eslint-config, stylelint-config, webpack-config, tailwind-config...)

## Delete
- Delete @studiometa/meta-configs

### CODE REVIEW

**Review by titouanmathis — CHANGES_REQUESTED** [review]
Thanks for the update! 👍 

Could you also include the following small fixes I had to make when creating a new project? 

- [x] Rename `config/assets.yaml` to `config/assets.yml` (there is a warning with the `.yaml` extension)
- [x] Remove the obsolete `front-page.js` asset from the `config/assets.yml` file as it is not used anymore
- [x] Remove the obsolete `acf-json/` folder in the theme 
- [x] Fix a PHPStan error from the `studiometa-plugin-disabler` MU Plugin with the following change:

```diff
-      $plugins_to_disable = explode( ',', $plugins_to_disable_raw );
+      $plugins_to_disable = array_filter( explode( ',', $plugins_to_disable_raw ) );
```

That would be great! 

You will also need to bump the package version number in the root `package.json` file 😉

**Comment by perruche** [comment]
> Thanks for the update! 👍
> 
> Could you also include the following small fixes I had to make when creating a new project?
> 
> * [x]  Rename `config/assets.yaml` to `config/assets.yml` (there is a warning with the `.yaml` extension)
> * [x]  Remove the obsolete `front-page.js` asset from the `config/assets.yml` file as it is not used anymore
> * [x]  Remove the obsolete `acf-json/` folder in the theme
> * [ ]  Fix a PHPStan error from the `studiometa-plugin-disabler` MU Plugin with the following change:
> 
> ```diff
> -      $plugins_to_disable = explode( ',', $plugins_to_disable_raw );
> +      $plugins_to_disable = array_filter( explode( ',', $plugins_to_disable_raw ) );
> ```
> 
> That would be great!
> 
> You will also need to bump the package version number in the root `package.json` file 😉

Should i add the fix for "Allow WordPress to detect HTTPS when used behind a reverse proxy or a load balancer" inside `wp-config-sample.php` ? 

`studiometa-plugin-disabler` is not inside this repo, but will do on the other one

**Comment by titouanmathis** [comment]
> `studiometa-plugin-disabler` is not inside this repo, but will do on the other one

My bad, it does not appear in this hotfix as it has not yet been released. It can be found on the `develop` branch. Maybe we could create a release to include the latest updates from `develop`?

> Should i add the fix for "Allow WordPress to detect HTTPS when used behind a reverse proxy or a load balancer" inside wp-config-sample.php ?

Yes, good idea! I forgot about this one, but it can be annoying to debug infinite https:// redirection loops (😅), so having it would be great.

**Comment by perruche** [comment]
> > `studiometa-plugin-disabler` is not inside this repo, but will do on the other one
> 
> My bad, it does not appear in this hotfix as it has not yet been released. It can be found on the `develop` branch. Maybe we could create a release to include the latest updates from `develop`?

Oh right, this will cause issue with the version number ? Release should be beta9, and hotfix beta10 then ?

**Comment by titouanmathis** [comment]
> Oh right, this will cause issue with the version number ? Release should be beta9, and hotfix beta10 then ?

Well, we can finish this hotfix, publish the `1.0.0-beta.9` version and then open a new release for `1.0.0-beta.10`. Would this be okay for you?

**Comment by perruche** [comment]
> > Oh right, this will cause issue with the version number ? Release should be beta9, and hotfix beta10 then ?
> 
> Well, we can finish this hotfix, publish the `1.0.0-beta.9` version and then open a new release for `1.0.0-beta.10`. Would this be okay for you?

yeah ! im on it !

File: template/bin/source/wp-config-sample.php [hunk 1]
```diff
@@ -122,5 +122,13 @@
 /* Set default theme */
 define( 'WP_DEFAULT_THEME', '<%= slug %>' );
 
+/**
+ * Allow WordPress to detect HTTPS when used behind a reverse proxy or a load balancer
+ * @see https://codex.wordpress.org/Function_Reference/is_ssl#Notes
+ */
+if ( isset( $_SERVER['HTTP_X_FORWARDED_PROTO'] ) && $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https' ) {
```
Comment (titouanmathis) [inline template/bin/source/wp-config-sample.php]: **question:** Maybe we should use a Yoda condition here? 🤔

**Review by titouanmathis — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@studiometa/create-wordpress-project",
-  "version": "1.0.0-beta8",
+  "version": "1.0.0-beta9",
```
Comment (titouanmathis) [inline package.json]: **issue:** Change `beta9` to `beta.9`

The beta version format for NPM packages is `vX.X.X-beta.X`.

**Pull request merged**